presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableProperties.java

```diff
@@ -73,6 +76,11 @@ public IcebergTableProperties(IcebergConfig icebergConfig)
                         "Format version for the table",
                         null,
                         false))
+                .add(integerProperty(
+                        CONCURRENCY_RETRIES,
+                        "Determines the number of attempts in case of concurrent upserts",
```

Comment:
Concurrent updates

Reply:
I think it is valid in case of inserts as well. Ultimately this boils down to some writer finding the metadata file updated, as compared to what it saw in the beginning, when it tries to commit.

Reply:
By update, I am trying to convey a more general meaning that upset, which may include deletes. Feel free to use any language that encapsulates this, but I feel upsert does not.

Reply:
I can probably make it upserts and deletes then.